Makefile

```diff
@@ -88,30 +88,9 @@ endif
 
 export DBUILD_ARGS=--build-arg DBUILD_DATE=${DBUILD_DATE} --build-arg DBUILD_REPO_URL=${DBUILD_REPO_URL} --build-arg DBUILD_SITE_URL=${DBUILD_SITE_URL} --build-arg ARCH=${ARCH}
```

Comment:
The ARCH build-arg is not required as it is not used in any dockerfile like https://github.com/openebs/dynamic-localpv-provisioner/blob/develop/buildscripts/provisioner-localpv/Dockerfile